### ISSUE TITLE
Add brakeman security checker and fix errors and warnings

### ISSUE THREAD

**Issue by compwron** [doubleunion/arooo#396]
This issue is a replacement for 
https://github.com/doubleunion/arooo/issues/106
https://github.com/doubleunion/arooo/issues/107

https://brakemanscanner.org

```

== Overview ==

Controllers: 18
Models: 7
Templates: 46
Errors: 0
Security Warnings: 8

== Warning Types ==

Attribute Restriction: 3
Cross-Site Request Forgery: 1
Dangerous Send: 1
Mass Assignment: 3

== Warnings ==

Confidence: High
Category: Attribute Restriction
Check: ModelAttributes
Message: Mass assignment is not restricted using `attr_accessible`
File: app/models/vote.rb
Line: 1

Confidence: High
Category: Dangerous Send
Check: Send
Message: User controlled method execution
```

**Comment by sbpipb** [comment]
hello! I'm interested to take this on. Please confirm if this is still valid and available. Thank you very much.

**Comment by compwron** [comment]
Yep it's still available! Please do :)

**Comment by anaulin** [comment]
Unassigned this issue so that it is clearer that it is up for grabs, since there's been no activity for a while. @sbpipb if you still want to work on this, you're very welcome! (Let me know if you need help/support with it.)

**Comment by ircarreira** [comment]
Addressed by #510 

@anaulin @compwron please let me know what you think
